tools/generators/lib/GeneratorCommon/src/ParsableCommand+Extensions.swift

```diff
@@ -0,0 +1,34 @@
+import ArgumentParser
+import Foundation
+
+extension ParsableCommand {
+    public static func parseAsRootSupportingParamsFiles() async {
```

Comment:
Is there any need to exercise this and explicitly assert in a test that `parseAsRootSupportingParamsFiles` is parsing things correctly?

Reply:
Since it's pretty small and understandable, and to make it testable would be a pain, and since this made my generator call that used a params file work, I think that's good enough for me.

Reply:
I'm fine with someone else adding a test though. I just realized we can create temp files like how we do for `ResolveSymlinksTests`, and we can create a mock `ParsableCommand`.